vajram/vajram-codegen/src/main/java/com/flipkart/krystal/vajram/codegen/models/ParsedVajramData.java

```diff
@@ -51,10 +52,38 @@ public static Optional<ParsedVajramData> fromVajram(VajramInfo vajramInfo, Utils
             vajramInfo.responseType()));
   }
 
+  public static void handleDuplicateDependencyBySameResolverException(List<ExecutableElement> methods){
+    HashMap<String, HashMap<String,Boolean>> lookUpMap = new HashMap<>();
+    for(ExecutableElement method: methods){
+      String depName = method.getAnnotation(Resolve.class).depName();
+      String [] depInputs = method.getAnnotation(Resolve.class).depInputs();
```

Comment:
We have to handle the case that Resolve annotation is missing

Reply:
for no @resolver annotation u mean .. that we will do in separate task .... does that work??